### ISSUE TITLE
Doc Update for Mac

### ISSUE THREAD

**Issue by Steve-Groner** [NativeScript/docs-v7#1700]
### Please, tell us what's the problem?
[ ] A typo
[ ] Wrong documentation
[X] Improvement of existing article
[ ] Missing documentation
[ ] New article needed

### Please, tell us more details
On this page https://docs.nativescript.org/start/ns-setup-os-x after step 5, step six uses pip to install six.  However on a default install of Mac OSx pip is not yet installed and needs to be installed using.

`sudo easy_install pip`

This will allow the user to use pip to install 'six'

**Comment by etabakov** [comment]
Thanks for the heads up @Steve-Groner. Feel free to open a PR with this suggestion. We will be happy to review it.

**Comment by mediaformat** [comment]
[X] Improvement of existing article

On Mojave OS, ran into a few issues.

**Step 3.5. Install Cocoapods**
```
$ pod setup
'expand_path': user ~ doesn't exist (ArgumentError)
```

~~**Step 7 To check if your system is configured properly, run the following command.**~~
```
$ tns doctor
-bash: tns: command not found
```
**Edit SOLVED Step 7** It turns out that in step 6.1 `npm i -g nativescript` was not installing in the right place . [This](http://michaelb.org/the-right-way-to-do-global-npm-install-without-sudo/) helped me fix it.
